test/js/specs/happy.js

```diff
@@ -12,34 +12,120 @@ describe('Happy Paths',options, ({driver,$,pageObjects}) => {
   const {documentSelection, welcome, documentUpload} = pageObjects
 
   //Welcome Screen Tests
-  const copy = welcome.copy()
+  const copyWelcome = welcome.copyWelcome()
 
   it('test website title', async () => {
-    await driver.get(localhostUrl)
+    driver.get(localhostUrl)
     const title = driver.getTitle();
     expect(title).to.equal('Onfido SDK Demo');
   })
 
   it('test welcome screen title', async () => {
     const welcomeTitleText = welcome.welcomeTitle.getText()
-    expect(welcomeTitleText).to.equal(copy["title"]);
+    expect(welcomeTitleText).to.equal(copyWelcome["title"]);
     welcome.welcomeTitle.isDisplayed()
   })
 
   it('test welcome screen subtitle', async () => {
     const welcomeSubtitleText = welcome.welcomeSubtitle.getText()
-    expect(welcomeSubtitleText).to.equal(copy["description_p_1"] + "\n" + copy["description_p_2"]);
+    expect(welcomeSubtitleText).to.equal(copyWelcome["description_p_1"] + "\n" + copyWelcome["description_p_2"]);
     welcome.welcomeSubtitle.isDisplayed()
   })
 
   it('test verify identity button', async () => {
     const verifyIdentityBtnText = welcome.primaryBtn.getText()
-    expect(verifyIdentityBtnText).to.equal(copy["next_button"]);
+    expect(verifyIdentityBtnText).to.equal(copyWelcome["next_button"]);
     welcome.primaryBtn.isDisplayed()
   })
 
   it('test footer is displayed', async () => {
     welcome.footer.isDisplayed()
+   })
+
+  //Document selection screen
+  const copyDocumentSelection = documentSelection.copyDocumentSelection()
+  const copyDocuments = documentSelection.copyDocuments()
+
+  it('test document selection title', async () => {
+    driver.get(localhostUrl)
+    welcome.primaryBtn.click()
+    const documentSelectionTitleText = documentSelection.title.getText()
+    expect(documentSelectionTitleText).to.equal(copyDocumentSelection["title"]);
```

Comment:
Here you have semicolon but nowhere else 🤔 ESLint?

Reply:
🤔 
`;` are used in all other `expect`ations

Reply:
It's a minor, however to me it seems a bit odd code-wise that some lines have semicolons, some not - code seems inconsistent. IMO there should be ESLint taking care of it + possible Prettier auto-formatting.

Reply:
Removed `;`